File: 0027-remove-element/0027-remove-element.py
class Solution(object):
    def removeElement(self, arr, val):
        n = len(arr)-1
        k = n
        for i in range(n, -1, -1):
            if val == arr[i]:
                arr[i], arr[k] = arr[k], arr[i]
                k -= 1
        return k+1
        

File: 0027-remove-element/0027-remove-element.cpp
class Solution {
public:
    int removeElement(vector<int>& a, int v) {
        int k=0;
        for(int i=0; i<a.size(); i++){
            if(a[i] != v) a[k++] = a[i];
        }
        return k;
    }
};